templates/etc/openssl/server_extfile.cnf.j2

```diff
@@ -0,0 +1,2 @@
+subjectAltName = DNS:{{ common_name }},IP:{{ansible_default_ipv4.address}},IP:127.0.0.1
```

Comment:
Would it make sense to add `DNS:localhost` too?

Reply:
Done